lab2.js

```diff
@@ -109,29 +162,94 @@ assert((new Human()).sayHello(new Klingon()) === "nuqneH",
 // Implement the following functions. Write at least 2
 // assertions for each one
 //*********************************************************
+
+///Here is my original answer that just sorts by the last letter.
+///But I think I wrote better code below that sorts by the last letter but
+///keeps looking at letters (if it matchs on any leter)
```

Comment:
Nice job seeing the limitations of one approach, and then coming up with a better way.